geode-core/src/main/java/org/apache/geode/internal/cache/FilterProfile.java

```diff
@@ -1170,10 +1170,10 @@ public void getLocalFilterRoutingForPutAllOp(DistributedPutAllOperation dpao,
       int size = putAllData.length;
       CqService cqService = getCqService(dpao.getRegion());
       boolean doCQs = cqService.isRunning()
-          && this.region != null /*
-                                  * && !(this.region.isUsedForPartitionedRegionBucket() ||
-                                  * (this.region instanceof PartitionedRegion))
-                                  */;
+      /*
```

Comment:
Can we just delete this commented out code?

Reply:
fixed